scripts/ubuntu_bionic_disco_dev_env_install.sh

```diff
@@ -0,0 +1,180 @@
+else
+    echo "** db $db_name already exists"
+fi
+
+echo "** create temporary virtenv to migrate bdd"
+if [ -d /tmp/venv_navitia_sql ]; then
+    rm -rf /tmp/venv_navitia_sql
+fi
+virtualenv /tmp/venv_navitia_sql
+. /tmp/venv_navitia_sql/bin/activate
+pip install -r "$navitia_dir"/source/sql/requirements.txt
+
+echo "** migrate navitia bdd with alembic"
+cd "$navitia_dir"/source/sql
+PYTHONPATH=. alembic -x dbname="postgresql://$db_owner:$db_user_password@localhost/$db_name" upgrade head
+deactivate
+rm -rf /tmp/venv_navitia_sql
+
+echo "** create jormun db "
+if ! sudo -i -u postgres psql -l | grep -q "^ jormungandr"; then
+    sudo -i -u postgres createdb jormungandr -O "$db_owner"
+    sudo -i -u postgres psql -c "create extension postgis; " jormungandr
+else
+    echo "** db jormungandr already exists"
+fi
+
+echo "** create temporary virtenv to upgrade jormun bdd"
+if [ -d /tmp/venv_tyr ]; then
+    rm -rf /tmp/venvtyr
+fi
+virtualenv /tmp/venv_tyr
+. /tmp/venv_tyr/bin/activate
+pip install -r "$navitia_dir"/source/tyr/requirements_dev.txt
+
+echo "** migrate jormungandr bdd with alembic"
+cd "$navitia_dir"/source/tyr
+PYTHONPATH=.:../navitiacommon/ TYR_CONFIG_FILE=default_settings.py ./manage_tyr.py db upgrade
+cd -
+deactivate
+rm -rf /tmp/venv_tyr
+
+echo "**"
+echo "**"
+echo "**"
+echo "** the installation is finished"
+echo "**"
+echo "**"
+echo "                     WELCOME TO"
+echo ""
+echo "##    ##    ###    ##     ## #### ######## ####    ###"
+echo "###   ##   ## ##   ##     ##  ##     ##     ##    ## ##"
+echo "####  ##  ##   ##  ##     ##  ##     ##     ##   ##   ##"
+echo "## ## ## ##     ## ##     ##  ##     ##     ##  ##     ##"
+echo "##  #### #########  ##   ##   ##     ##     ##  #########"
+echo "##   ### ##     ##   ## ##    ##     ##     ##  ##     ##"
+echo "##    ## ##     ##    ###    ####    ##    #### ##     ##"
+echo ""
+echo "you are ready to push PRs"
+echo ""
+echo ""
+echo "**********************************************"
+echo "* !!! WARNING !!!"
+echo "* You should probably log out or reboot because of docker."
+echo "* Please test if docker run correctly with :"
+echo "* docker run hello-world"
+echo "**********************************************"
+echo ""
+echo ""
+echo "1. clone navitia"
+echo "$ git clone https://github.com/CanalTP/navitia.git"
+echo ""
+echo "for convenience reason, some submodule links are in ssh (easier to push)"
+echo "it is however thus mandatory for the user to have a github access"
+echo "Can change the ssh links to https with :"
+echo "$ sed -i 's,git\@github.com:\([^/]*\)/\(.*\).git,https://github.com/\1/\2,' .gitmodules"
+echo ""
+echo "2. compile navitia"
+echo "$ mkdir build"
+echo "$ cd build"
+echo "$ cmake -DCMAKE_BUILD_TYPE=RELEASE navitia_source_dir"
+echo "$ make -j"
+echo ""
+echo "3. install ED binaries [fusio2ed, ed2nav, ...] for eitri"
+echo "$ make install (as a super user)"
+echo "or"
+echo "create a sym link to recognize binaries"
```

Comment:
The point is to have them in $PATH (so it's possible add it directly)

Reply:
Yes exactly, and it is more clear ✌️